docs/existing-aws-infrastructure.md

```diff
@@ -38,10 +38,11 @@ spec:
   networkSpec:
     vpc:
       id: vpc-0425c335226437144
+    subnets:
+    - id: subnet-0261219d564bb0dc5
+    - id: subnet-0fdcccba78668e013
```

Comment:
These are optional though right?

Reply:
Not any more with this change if you have an unmanaged vpc.  If you have an umanaged vpc then you need to supply the subnets to use instead of it trying to determine / use all subnets in the vpc.....so it becomes bring your own subnets and explicitly tell capa about them.

Reply:
Marking resolved as per Slack and CAPA meeting of 2020/06/15.